lib/fs.js

```diff
@@ -1770,6 +1770,8 @@ ReadStream.prototype.close = function(cb) {
   close();
 
   function close(fd) {
+    if (fd === undefined && self.fd === null)
+      return;
```

Comment:
Why not emit an error after it is already closed? Does calling `.close()` twice on a stream count as a programmer error? If yes, perhaps it shouldn't be thrown under the rug.